### PR TITLE
Feat/become a parent notifications

### PR DESCRIPTION
**Pull request by two-first-names**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/features/notifications/notification-become-a-parent/notification-become-a-parent.component.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        this.parentRequestsService.getParentRequest(this.notification.typUid, this.establishmentService.establishmentId).subscribe(request => {
+          this.status = request.status;
+        });
+      })
+    );
+    this.setNotificationViewed(this.notificationUid);
+  }
+
+  /**
+   * Function used to set nothification as read
```
Comment (aaron-russell) [inline src/app/features/notifications/notification-become-a-parent/notification-become-a-parent.component.ts:48]: Spelling mistake, notifications

File: src/app/features/notifications/notification-become-a-parent/notification-become-parent.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    component.debug();
+
+    const message = component.getByTestId('message');
+    const type = component.getByTestId('type');
+
+    expect(message.innerText).toBe('Your request to become a parent has been accepted.');
+    expect(type.innerText).toBe('Become a parent organisation');
+  });
+
+  it('should display the correct message when the request is approved', async () => {
```
Comment (aaron-russell) [inline src/app/features/notifications/notification-become-a-parent/notification-become-parent.component.spec.ts:71]: Think you mean rejected?

File: src/app/features/notifications/notification-become-a-parent/notification-become-a-parent.component.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+@Component({
+  selector: 'app-notification-become-a-parent',
+  templateUrl: './notification-become-a-parent.component.html',
+  providers: [Overlay],
+})
+export class NotificationBecomeAParentComponent implements OnInit, OnDestroy {
+  public workplace: Establishment;
+  public notification;
+  private subscriptions: Subscription = new Subscription();
```
Comment (aaron-russell) [inline src/app/features/notifications/notification-become-a-parent/notification-become-a-parent.component.ts:20]: Generally, try and group private and public together

**Comment by aaron-russell** [comment]
Obviously, you have failing tests too

File: src/app/features/notifications/notification-become-a-parent/notification-become-parent.component.spec.ts (hunk 2 above)
Comment (two-first-names) [reply src/app/features/notifications/notification-become-a-parent/notification-become-parent.component.spec.ts]: Yup. :D

**Pull request merged**